src/EnergyPlus/SystemAvailabilityManager.hh

```diff
@@ -84,6 +84,9 @@ namespace SystemAvailabilityManager {
 	extern int const HybridVentMode_Enth; // Enthalpy control
 	extern int const HybridVentMode_DewPoint; // Dew point control
 	extern int const HybridVentMode_OA; // Outdoor air control
+	extern int const HybridVentMode_OperT80; // Operative temperature control with 80% acceptability limits
```

Comment:
@Myoldmopar Do these need to be extern?  They are never used outside SystemAvailabilityManager.  Lots of constants like this in SystemAvailabilityManager.